services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = "",
+        last_name: str = "",
+) -> None:
+
+    get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name,
+    )
+
+
+def get_user(user_id: int) -> User:
+
+    return get_user_model().objects.get(pk=user_id)
```

Comment:
Consider handling the case where the user with the given `user_id` does not exist. You can use `get_object_or_404` or handle the `get_user_model().DoesNotExist` exception to provide a more user-friendly error message.